daemon/containerd/image_list.go

```diff
@@ -337,12 +337,15 @@ func (i *ImageService) imageSummary(ctx context.Context, img images.Image, platf
 				"contentSize": contentSize,
 			}).Debug("content size calculation mismatch")
 
-			totalSize += contentSize - prevContentSize
+			mfstSummary.Size.Content = contentSize
+			d := contentSize - prevContentSize
+			totalSize += d
+			mfstSummary.Size.Total += d
```

Comment:
This code is hard for me to parse (incl. above).

Could we rename some things (such as `prevContentSize` and `contentSize`? Maybe we can call the image specific content size `singlePlatformSize` or something, and then we don't need to have `prevContentSize`?

Reply:
Renamed and added some clarification comments, let me know if that makes it better.

Reply:
ty!